junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/MethodSource.java

```diff
@@ -45,9 +45,9 @@
 public @interface MethodSource {
 
 	/**
-	 * The names of the test class methods to use as sources for arguments; must
-	 * not be empty.
+	 * The names of the test class methods to use as sources for arguments;
+	 * leave empty if the source method has the same name as the test method.
 	 */
-	String[] value();
+	String[] value() default "";
```

Comment:
A simple and small addition of `default ""` ... triggers so much more lines and feed-back loops. Great! 👍